src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -1188,18 +1188,21 @@ public void OpenFileFromPath(string xmlPath, bool forceManualExecutionMode = fal
                         OnWorkspaceOpening(xmlDoc);
 
                         // TODO: #4258
-                        // Remove this ResetEngine call when multiple home workspaces is supported.
-                        // This call formerly lived in DynamoViewModel
-                        ResetEngine();
-
-                        // TODO: #4258
                         // The following logic to start periodic evaluation will need to be moved
                         // inside of the HomeWorkspaceModel's constructor.  It cannot be there today
                         // as it causes an immediate crash due to the above ResetEngine call.
```

Comment:
It seems like `crash due to the above ResetEngine call` is no longer valid with this fix, perhaps reword/remove the comments?